### ISSUE TITLE
On accessing the FormItem inside Form, the associate label name is not announced to the screen reader user

### ISSUE THREAD

**Issue by C5362575** [SAP/ui5-webcomponents#9002]
### Bug Description

On accessing the FormItem, the associate label name is not announced to the screen reader Screen Reader: JAWS Version 2024.2403.3
A sample code is shared below for rederence
```
import {
  Form,
  FormItem,
} from "@ui5/webcomponents-react";

 <Form>
        <FormItem>
          label="labelText"
          aria-label="labelText"
        >
          <CustomComponent />
        </FormItem>
  </Form>
```

### Affected Component

Form,FormItem

### Expected Behaviour

All the contents and the information should be conveyed to the user, the corresponding label and group label should be read out, when each & every UI is accessed.

Relationships and information conveyed by the presentation (such as labels) should be identified by assistive technologies so user can connect the accessed UI with associated label.

 

### Steps to Reproduce

1.
2.
3.
...

### Priority

Medium

### UI5 Web Components Version

1.19.0

### Browser

Chrome

### Operating System

Windows 11 Enterprise, 23H2

### Declaration

- [X] I’m not disclosing any internal or sensitive information.
